config/routes.rb

```diff
@@ -151,8 +151,8 @@
   match "/500", to: "errors#internal_server_error", via: :all
 
   # If parameters are used that are the same as those in the search form, pagination with kaminari will break
-  match "teaching-jobs-in-:location",
-        to: "vacancies#index", as: :location, via: :get,
+  match "teaching-jobs-in-:location_facet",
+        to: "vacancies#index", as: :location_facet, via: :get,
```

Comment:
```
  match "teaching-jobs-in-:location_facet",
        to: "vacancies#index", as: :location, via: :get,
        constraints: ->(request) { LocationPolygon.include?(request.params[:location_facet]) }
```

Reply:
Thanks - this solves it without creating a second bug!